recursion/exponent.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
Specifically the part where it says the functions should be self contained and not in a main package.